Formula/cling.rb

```diff
@@ -0,0 +1,34 @@
+class Cling < Formula
+  desc "interactive C++ interpreter"
+  homepage "https://cdn.rawgit.com/root-mirror/cling/master/www/index.html"
+  url "https://github.com/vgvassilev/llvm.git", :using => :git, :tag => "cling-patches-r274612", :revision => "bc990e2aa9448cfea850efc31fda4201718f4e0f"
+  version "0.3"
```

Comment:
This shouldn't be necessary once the `url` is updated to https://github.com/vgvassilev/cling/archive/v0.3.tar.gz